ChangeLog.d/stdout-macro.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * Duplicate file descriptor into STDOUT_FILENO instead of replacing stdout
```

Comment:
Thanks for adding the ChangeLog entry. Generally, it is more valuable to users to describe context and symptoms of the problem rather than the mechanics of the fix. Suggestion:
```
    * Fix bug in redirection of unit test outputs which previously lead to some tests 
      getting stuck on NetBSD. Reported in #2311 and fix contributed in #3528 by XXX.
```
Feel free to be more specific and to add your name here as well, if you wish.